admin/javascript/SSNotice.js

```diff
@@ -0,0 +1,205 @@
+/**
+* File: SSNotice.js
+*
+* Replace the old statusMessage/errorMessage
+* with a custom library: ss.notice
+*
+* Allows for concistent notices/message display throughout the CMS
+* via simple JavaScript function calls:
+* - ss.notice.message(text, [options]);
+* - ss.notice.success(text, [options]);
+* - ss.notice.warning(text, [details], [options]);
+* - ss.notice.error(text, [details], [options]);
+**/
+(function($, root) {
+
+	// local working version of the library object
+	var lib = {};
+
+	/**
+	 * Library settings
+	 */
+	lib.settings = {
+		parentSelector: '.cms-content-fields',
+		$notice: $('<p class="message"/>'),
```

Comment:
Fair enough. For classes, this is configurable through the `options.type`, similar thing could be done for the node type too.

Hidden el don't make sense to me since it would involve more DOM manipulation. I guess we could have a stripped down base notice in a JS template loaded via SS Requirements so dev should be able to override it if needed and it give us an easy place to edit the markup?